examples/nodejs_alternative_builder/flake.nix

```diff
@@ -16,7 +16,7 @@ Building 'Prettier@2.4.1'.
     src,
   }:
     (dream2nix.lib.makeFlakeOutputs {
-      systems = ["x86_64-linux"];
+      systems = ["x86_64-linux" "aarch64-darwin"];
```

Comment:
As you see in the test log, we can't currently test this. So we either need to exclude it from tests or get https://github.com/nix-community/infra/issues/493 finished before merging if we include this in the example.

We could ofc just bump the flake lock in a separate PR if tests pass.